docs/modules/ROOT/pages/vshn-managed/nextcloud/create.adoc

```diff
@@ -0,0 +1,57 @@
+= Create a Nextcloud service
+
+Apply the following object on your namespace, as specified by its YAML description.
+
+.Example to create a Nextcloud instance
+[source,yaml]
+----
+apiVersion: vshn.appcat.vshn.io/v1
+kind: VSHNNextcloud
+metadata:
+  name: nextcloud-app1-prod # <1>
+  namespace: prod-app # <2>
+spec:
+  parameters:
+    service:
+      version: "23" # <3>
+      defaultInternalDB: true # <4>
```

Comment:
The name of this field is confusing. If it contains `default` I would assume this is on by default, but it's false by default.

Maybe `useInternalDB`?

Reply:
We had some issues with the naming and came up with `defaultInternalDB`. I guess your point is valid and can go with your suggestion.

Reply:
Changed it to useExternalPostgreSQL